x-pack/plugins/security_solution/public/management/common/constants.ts

```diff
@@ -39,6 +39,8 @@ export const MANAGEMENT_STORE_HOST_ISOLATION_EXCEPTIONS_NAMESPACE = 'hostIsolati
 export const MANAGEMENT_PAGE_SIZE_OPTIONS: readonly number[] = [10, 20, 50];
 export const MANAGEMENT_DEFAULT_PAGE = 0;
 export const MANAGEMENT_DEFAULT_PAGE_SIZE = 10;
+export const MANAGEMENT_DEFAULT_SORT_ORDER = 'desc';
+export const MANAGEMENT_DEFAULT_SORT_FIELD = 'created_at';
```

Comment:
just want to confirm - these are actually the defaults we are using across all pages correct? normally we don't set sort field/order and I think ES defaults it to `updated_at` (mabye?).

Reply:
Yeah, we're using these defaults everywhere except for TAs. TAs sort by `name` in an `asc` order. Not sure why that is, but perhaps we forgot to update it for TAs?

And yes, it defaults to `updated_at` by `asc` order.

Reply:
yeah, TA were weird. That will just be fixed once we move that page to the new `<ArtifactListPage/>` component. 
Thanks for the response.